pkg/manager/manager.go

```diff
@@ -1890,6 +1891,12 @@ func (m *jobManager) CheckValidJobConfiguration(req *JobRequest) error {
 	return nil
 }
 
+var allowedRegistries = []string{
+	"registry.ci.openshift.org",
+	"quay.io",
+	"registry.build06.ci.openshift.org",
```

Comment:
There are 11 possible build clusters now...
https://docs.ci.openshift.org/docs/getting-started/useful-links/#clusters

Reply:
yup, then we don't have to fetch the build* ones dynamically. I split it up by static name and regex matching patterns.